services/resource-usage-tracker/tests/unit/with_dbs/conftest.py

```diff
@@ -105,6 +113,35 @@ def _creator(**overrides) -> dict[str, Any]:
     return _creator
 
 
+@pytest.fixture
+def random_resource_tracker_credit_transactions(
+    faker: Faker,
+) -> Callable[..., dict[str, Any]]:
+    def _creator(**overrides) -> dict[str, Any]:
+        data = {
+            "product_name": "osparc",
+            "wallet_id": faker.pyint(),
+            "wallet_name": faker.word(),
+            "pricing_plan_id": faker.pyint(),
+            "pricing_unit_id": faker.pyint(),
+            "pricing_unit_cost_id": faker.pyint(),
+            "user_id": faker.pyint(),
+            "user_email": faker.email(),
+            "osparc_credits": -abs(faker.pyfloat()),
+            "transaction_status": choice(["BILLED", "PENDING", "NOT_BILLED"]),
```

Comment:
I would parametrize this to spin up more tests. I thought this fixture would be used in a loop.

Reply:
Yes when the need comes I will parametrize and reuse.